### DISCUSSION TITLE
Cluster Site Relation format?

### DISCUSSION THREAD

**Discussion by tkn2023** [bb-Ricardo/netbox-sync#382] [General]
I have a long list for the cluster_site_relation option in the settings.ini.
It is working so far. But I encountered an issue with 2 of my sites. They both start with the same 3 letters but the 4th one is different so I have this as follows in the settings.ini:
`cluster_site_relation = abcd* = ABCDEFFG, abce* = ABCEGHAALS`
During the script execution I receive:
`DEBUG2: Trying to find site name for cluster 'abce/abce005'
DEBUG2: Found a matching cluster_site_relation 'ABCDEFFG' (abcd*) for abce/abce005
DEBUG2: Found a matching cluster_site_relation 'ABCEGHAALS' (abce*) for abce/abce005
DEBUG: Found 2 matches for abce/abce005 in cluster_site_relation. Using first on: ABCDEFFG `

Anything I can do differently here?

**Comment by tkn2023** [comment]
Found the problem. I needed to  add .*
So instead of abcd* it has to be abcd.*
Like I said I am pretty new to this.

**Comment by bb-Ricardo** [comment]
Hi,

Yes it uses regular expressions, therefore you need the dot.
